### DISCUSSION TITLE
关于RSS3的问题与建议

### DISCUSSION THREAD

**Discussion by oblank** [NaturalSelectionLabs/RSS3-Protocol#27] [General]
**问：** 这是打算丰富RSS的数据结构吗？主要有哪些改进性？只看 md 未必能理解你们想解决的问题。

**建议:**  在 `RSS3Item` 里增加过滤相关的东西，在我做的某东西里叫它“过滤器”，譬如：一个论坛的贴子有`回复数`，一个文章有`点击量`，一个ProductHunt的内容有`投票数`等，当然可过滤的不一定是数字的，也可能是文本性的，需要类型标识。

之所以会有过滤器的东西出来是因为 RSS 将所有信息平权化了，而实际上内容本身是有优劣好坏之分的，过滤器一定程度上可以让用户管理自己的订阅内容的质量，或许可以结合`tag`来做

另，我之前也思考过对RSS的结构进行一些升级，我叫它`RSS Plus` :)

**Comment by DIYgod** [comment]
这些交互内容都是放在 context 里的，有过滤需求可以自行根据这个过滤

**Comment by sun158158** [comment]
Personally, I think the biggest value of community autonomy is real decentralization, and then you have to issue your own tokens to create a high-quality autonomous community, which can also empower the project.
Really want to get your reward, a good project, 
wallet address is 0x516fd29c0fb569a749c0c244bd59484c905d480f thank you
